server/handler/assignments.go

```diff
@@ -96,3 +96,45 @@ func SaveAssignment(repo *repository.Assignments) RequestProcessFunc {
 		return serializer.NewCountResponse(1), nil
 	}
 }
+
+// GetFilePairAnnotations returns a function that returns a *serializer.Response
+// with the Annotation results for the given File Pair and Experiment IDs
+func GetFilePairAnnotations(repo *repository.Assignments) RequestProcessFunc {
+	return func(r *http.Request) (*serializer.Response, error) {
+		experimentID, err := urlParamInt(r, "experimentId")
+		if err != nil {
+			return nil, err
+		}
+
+		pairID, err := urlParamInt(r, "pairId")
+		if err != nil {
+			return nil, err
+		}
+
+		assignments, err := repo.GetByExperimentPair(experimentID, pairID)
```

Comment:
hm... it's not `assignments`, is it?

Reply:
Yes, those come from the assignments table

Reply:
ah. got you. thanks!